### PR TITLE
User page - badges and photo overlay (looks like a toughy) 

### PR DESCRIPTION
**Pull request by arfacamble**

### CODE REVIEW

File: app/assets/stylesheets/pages/_user_profile.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .profile-details {
+  background-color: #F4F4F4;
```
Comment (A-Marriott) [inline app/assets/stylesheets/pages/_user_profile.scss:2]: use this variable? $light-gray

Comment (todd-neale) [reply app/assets/stylesheets/pages/_user_profile.scss:2]: will change

File: app/assets/stylesheets/pages/_user_profile.scss [hunk 2]
```diff
@@ -17,37 +18,128 @@
...
+
+
+
+
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (todd-neale) [reply app/assets/stylesheets/pages/_user_profile.scss]: Spacey

**Pull request merged**